lib/optimize/InnerGraphPlugin.js

```diff
@@ -185,31 +185,27 @@ class InnerGraphPlugin {
 					/** @type {WeakMap<{}, TopLevelSymbol>} */
 					const statementWithTopLevelSymbol = new WeakMap();
 					parser.hooks.preStatement.tap("InnerGraphPlugin", statement => {
-						const innerGraphState = InnerGraph.getState(parser.state);
-						if (!innerGraphState) return;
+						if (!InnerGraph.isEnabled(parser.state)) return;
 
 						if (parser.scope.topLevelScope === true) {
 							if (statement.type === "FunctionDeclaration") {
-								const { innerGraph } = innerGraphState;
 								const name = statement.id ? statement.id.name : "*default*";
 								parser.defineVariable(name);
-								const fn = new TopLevelSymbol(name, innerGraph);
+								const fn = InnerGraph.createTopLevelSymbol(parser.state, name);
 								parser.tagVariable(name, topLevelSymbolTag, fn);
```

Comment:
createTopLevelSymbol always occurs together with `parser.tagVariable`. 

-> `const fn = InnerGraph.tagTopLevelSymbol(parser, name)`

Make it check for already existing TopLevelSymbol with `getTagData` so `tagVar` can also use it